src/bereikbaarheid/resources/utils.py

```diff
@@ -127,7 +127,7 @@ def refresh_materialized(db_table: str):
 def convert_to_date(date: str = None) -> datetime:
     """Convert string format %d/%m/%y %H:%M or %Y-%m-%d %H:%M:%S.%f to datetime"""
 
-    formats_allowed = ["%d/%m/%y %H:%M", "%Y-%m-%d %H:%M:%S.%f"]
+    formats_allowed = ["%d/%m/%y %H:%M", "%Y-%m-%d %H:%M:%S.%f", "%d-%m-%Y %H:%M","%Y-%m-%d %H:%M:%S"]
```

Comment:
`formats_allowed[1]` en `formats_allowed[3]` zijn volgens mij hetzelfde. Moet een van de twee hoofdletters `Y` een kleine letter `y` zijn?

En moeten er nog tests bij?

Reply:
formats_allowed[1] en formats_allowed[3] verschillen in de milliseconden. Dit komt doordat de export de locale settings meeneemt (lijkt het) daarom beiden maar opgenomen zodat de import er mee kan werken.